cc/resources/display_resource_provider.cc

```diff
@@ -28,6 +29,7 @@ DisplayResourceProvider::DisplayResourceProvider(
 DisplayResourceProvider::~DisplayResourceProvider() {
   while (!children_.empty())
     DestroyChildInternal(children_.begin(), FOR_SHUTDOWN);
+  resource_sk_image_.clear();
```

Comment:
How do you think whether a Castanets flag is needed for every modification or not?

Reply:
I thought, the changes in ScopedReadLockSkImage is generic. So didnt add build flag.
I have added build flags accordingly in latest patch set.